internal/commands/app/diff.go

```diff
@@ -97,10 +97,12 @@ func (cmd *CommandDiff) Handler(profile *user.Profile, ui terminal.UI, clients c
 	}
 
 	if cmd.inputs.IncludeDependencies {
-		uploadPath, err := local.PrepareDependencies(app, ui)
+		uploadDependencies, err := local.FindAppDependencies(app.RootDir)
```

Comment:
[naming nit] maybe we just call this `localDependencies` or `appDependencies` now